scvi/external/stereoscope/_model.py

```diff
@@ -224,6 +233,33 @@ def get_proportions(self, keep_noise=False) -> np.ndarray:
             index=self.adata.obs.index,
         )
 
+    def get_scale_for_ct(
+        self,
+        y: np.ndarray,
+    ) -> np.ndarray:
+        r"""
+        Return the scaled parameter of the NB for the list of cell type provided.
+
+        Parameters
+        ----------
+        y
+            numpy array containing the list of cell types
+        Returns
+        -------
+        gene_expression
```

Comment:
I'm not sure how to use this function from the docstring. An examples section in the docstring might help.